src/Diagnostics.RuntimeHost/Controllers/DiagnosticControllerBase.cs

```diff
@@ -362,6 +365,7 @@ protected async Task<IActionResult> PublishPackage(Package pkg)
             }
 
             await _sourceWatcherService.Watcher.CreateOrUpdatePackage(pkg);
+            await storageWatcher.CreateOrUpdatePackage(pkg);
```

Comment:
Can we make these two calls concurrent? Also what is your intention in the case of failure from storage watcher? Will it silently fail until we flip the switch or will you fail the API even if git hub watcher succeeds?

Reply:
So we cannot make the calls concurrent because we want to store the GitHub SHA in the table so storage watcher should execute after GitHub watcher finishes. 

Also, I have now coded it to be silently failing while logging the exceptions. May be once we are fully on new system we can fail the API

Reply:
What happens in the case when it is national clouds?

Reply:
@hforeste I have added an empty class that gets injected for national clouds. Please review the changes again.